sdk/Directory.Build.props

```diff
@@ -3,5 +3,10 @@
     <IsMgmtProject Condition="'$(IsMgmtProject)' == '' and $(MSBuildProjectName.Contains('.Management.'))">true</IsMgmtProject>
     <IsDataPlaneProject Condition="'$(IsDataPlanProject)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Azure.')) and '$(IsMgmtProject)' != 'true'">true</IsDataPlaneProject>
   </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
Why are these here? We should be converting these to the data-plane style projects. If we end up needing some of these properties for some reasons they should go into the eng/Directory.Build.Data.targets|props files.

Reply:
Will update this.